Sources/StreamChatUI/ChatMessageList/Attachments/GalleryAttachmentViewInjector.swift

```diff
@@ -37,10 +37,20 @@ open class _GalleryAttachmentViewInjector<ExtraData: ExtraDataTypes>: _Attachmen
         
         contentView.bubbleView?.clipsToBounds = true
         contentView.bubbleContentContainer.insertArrangedSubview(galleryView, at: 0, respectsLayoutMargins: false)
+        
+        // We need to apply corners to the left and right containers because the previewsContainerView
+        // is applying extra layout margins and the rounded corners wouldn't match the margins.
+        let leftCorners: CACornerMask = [.layerMinXMaxYCorner, .layerMinXMinYCorner]
+        galleryView.leftPreviewsContainerView.layer.maskedCorners = options.roundedCorners.intersection(leftCorners)
+        galleryView.leftPreviewsContainerView.layer.cornerRadius = 16
+        galleryView.leftPreviewsContainerView.layer.masksToBounds = true
+
+        let rightCorners: CACornerMask = [.layerMaxXMaxYCorner, .layerMaxXMinYCorner]
+        galleryView.rightPreviewsContainerView.layer.maskedCorners = options.roundedCorners.intersection(rightCorners)
+        galleryView.rightPreviewsContainerView.layer.cornerRadius = 16
+        galleryView.rightPreviewsContainerView.layer.masksToBounds = true
 
-        NSLayoutConstraint.activate([
-            galleryView.widthAnchor.pin(equalTo: galleryView.heightAnchor)
-        ])
+        galleryView.widthAnchor.pin(equalTo: galleryView.heightAnchor, multiplier: 1.32).isActive = true
```

Comment:
I am just curious 😅:   `multiplier: 1.32`  You have tried all options or is this some magic constant which just looks good or you figured out from figma designs by doing some math? Maybe worth to write comment for it :)

Reply:
Yes from figma. The size is 125 x 94,5; 125/94,5 = 1,32275